base/applications/regedit/childwnd.c

```diff
@@ -137,6 +137,7 @@ static void finish_splitbar(HWND hWnd, int x)
     GetClientRect(hWnd, &rt);
     g_pChildWnd->nSplitPos = x;
     ResizeWnd(rt.right, rt.bottom);
+    InvalidateRect(hWnd, &rt, FALSE);
```

Comment:
`draw_splitbar` does draw invert of pattern.
To erase it, we can call it again, or `InvalidateRect` like this.